lib/routes/account.js

```diff
@@ -1914,8 +1995,162 @@ module.exports = function (
             function () {
               reply({})
             },
+            function (err) {
+              reply(err)
+            }
+          )
+      }
+    },
+    {
+      method: 'GET',
+      path: '/recovery_email/emails',
+      config: {
+        auth: {
+          strategy: 'sessionTokenWithVerificationStatus'
+        },
+        response: {
+          schema: isA.array().items(
+            isA.object({
+              isVerified: isA.boolean().required(),
+              isPrimary: isA.boolean().required(),
```

Comment:
API nit: it probably makes sense to use the same fields here as we use in the existing `/recovery_email/status` response, which is `verified` rather than `isVerified`.